batch/batch.js

```diff
@@ -554,6 +554,28 @@ var canBatch = {
 		} else {
 			handler({});
 		}
+	},
+	/**
+	*/
+	debounce: function(handler){
```

Comment:
it is now non-enumerable, and has a test to verify. There was already a test to verify that it would run through multiple batches, as well as to verify that it skips a batch when the event is not triggered. I modified the tests to verify that the arguments were passed properly (they weren't).